Gopkg.toml

```diff
@@ -1,10 +1,6 @@
 # Force dep to vendor the code generators, which aren't imported just used at dev time.
 # Picking a subpackage with Go code won't be necessary once https://github.com/golang/dep/pull/1545 is merged.
 required = [
-  "k8s.io/code-generator/cmd/defaulter-gen",
```

Comment:
Are these code generators being removed b/c https://github.com/golang/dep/pull/1545 merged?

Reply:
Latest commit cleans up the entire file including the `required` stanza. We still need some stuff in `required` for non-imported code generation, but the subpackage note isn't relevant to us.